streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/controller/VariableController.java

```diff
@@ -91,7 +91,7 @@ public RestResponse variableList(@RequestParam Long teamId, String keyword) {
   @PostMapping("dependApps")
   @RequiresPermissions("variable:depend_apps")
   public RestResponse dependApps(RestRequest restRequest, Variable variable) {
-    IPage<Application> dependApps = variableService.dependAppsPage(variable, restRequest);
+    IPage<Application> dependApps = variableService.pageDependApps(variable, restRequest);
```

Comment:
how about `getPage`, as mentioned in the guide page, methods name would be better to start with a `verb`.